plugins/modules/postgresql_pg_hba.py

```diff
@@ -885,6 +884,7 @@ def main():
         if not module.check_mode:
             ret['msgs'].append('Writing')
             try:
+                backup_file = module.params['backup_file']
                 if pg_hba.write(backup_file):
```

Comment:
```suggestion
                if pg_hba.write(module.params['backup_file']):
```
a little bit shorter:)